frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -82,9 +140,8 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
       <ModalBody>
         <Trans t={t} ns="olm">
           This will remove Operator <strong>{{ name }}</strong> from{' '}
-          <strong>{{ namespace }}</strong>. Removing the Operator will not remove any of its custom
-          resource definitions or managed resources. If your Operator has deployed applications on
-          the cluster or configured off-cluster resources, these will continue to run and need to be
+          <strong>{{ namespace }}</strong>. If your Operator has deployed applications on the
+          cluster or configured off-cluster resources, these will continue to run and need to be
```

Comment:
Seems like a uxd question, @tlwu2013 ?

Reply:
Updated to include some verbiage about 'operand instances':
![image](https://user-images.githubusercontent.com/12733153/121247964-d4fa5500-c870-11eb-8032-9707c92d6941.png)

TBH, I don't know what the text "If your Operator deployed applications on the cluster or configured off-cluster resources, these will continue to run and require manual cleanup." is refering too?  @TheRealJon ?

Also, please see [recent changes](https://github.com/openshift/console/pull/8895) to the operator uninstall modal to also remove plug-ins.  This will add the following verbiage to the end of text if user has privleges to remove them:  "...require manual cleanup. _The console plugin provided by this operator will be disabled and removed._ "